.scripts/bump_version.py

```
@@ -222,28 +225,32 @@ def parse_existing_changelog_entry(content: str) -> dict:
+def generate_changelog_entry(
+    version, addon_path, changelog_message=None, existing_entry=None
+):
     """Generate a detailed changelog entry with clickable links."""
     entry_date = datetime.now().strftime("%Y-%m-%d")
     heading = f"## {version} ({entry_date})"
     entry = f"{heading}\n\n"
 
     repo_url = get_git_remote_url()
 
-    ignore_tags = [f"v{version}", f"{os.path.basename(addon_path.rstrip('/\\'))}-{version}"]
+    ignore_tags = [
```

Ruff (0.14.10)

Local variable `ignore_tags` is assigned to but never used

Remove assignment to unused variable `ignore_tags`

(F841)